openvox/install.md

```diff
@@ -5,16 +5,40 @@ subsection: openvox
 ---
 
 OpenVox package downloads are currently sponsored by [Overlook InfraTech](https://overlookinfratech.com).
-Following Perforce's suggestions, these are straight rebuilds of the original packages with no branding or name changes.
-Be aware that this will change as soon as the OpenVox pipelines are running.
+As of release 8.11, OpenVox is functionally equivalent to Puppet; the command names are the same, the configuration file paths are the same, etc.
+The major differences are in help text output, man pages, and so on.
+This means that you can continue to use all the commands, modules, tooling, etc that you're used to, but at this time *you cannot install both Puppet and OpenVox on the same system*.
+
+🚨 Before you enable and install the packages, be forewarned that these are still experimental packages only.
+We do not yet have a fully robust test pipeline, although that is our next priority.
+
+
+## Uninstalling Puppet
+
+We encourage you to try out OpenVox on a fresh test system, the way you would for any major system package.
+If you'd rather try it on an existing system or develop a migration process, then you will first have to uninstall Puppet.
+
+* If you're migrating from Puppet Enterprise you can use the `puppet-enterprise-uninstaller` script on each node as described in [their docs](https://www.puppet.com/docs/pe/latest/uninstalling.html).
+* If you're using the all-in-one packages such as `puppet-agent` or `puppetserver` from the `[apt|yum].puppet.com` repos, simply remove these packages.
```

Comment:
Do we have rpm/deb metadata that says it conflicts? Possibly even provides and automatically replaces the Perforce packages

Reply:
@nmburgan is working on the conflicts/replaces metadata. I *think* that will only be sufficient for the simple "replace the AIO package" case though. I don't think that removes unused dependencies. (but I could totally be wrong)

Reply:
@ekohl are you ok merging as-is? We can always come back and simplify if the metadata does indeed do the heavy lifting for us.

Reply:
merging as-is now, so we have install instructions that at least use the right names. We'll come back to it again after next release.